Assets/Scripts/PanZoom.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PanZoom : MonoBehaviour
+{
+	Vector3 touchFirst;
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    	if(Input.GetMouseButtonDown(0)){ //starts at the very beggining of the touch
+            touchFirst = Camera.main.ScreenToWorldPoint(Input.mousePosition);
+        }
+
+        if(Input.GetMouseButton(0)){ //continues panning during drag
```

Comment:
if you need a comment to explain what it does, extract it to a function with that name is usually a good go to